products/ceqr_survey/seeds/ceqr_variables.csv

```diff
@@ -1,2 +1,13 @@
-variable_name,variable_label,ceqr_category,type_ii_criteria
-zoning_districts,Zoning districts,Zoning,"R5-R10, R1-R4, or (C or M)"
\ No newline at end of file
+variable_type,variable_label,ceqr_category,type_ii_criteria_description
+zoning_districts,Zoning Districts,Zoning,"R5-R10, R1-R4, or (C or M)"
+zoning_districts,Special Coastal Risk Districts,Zoning,"Any of 3 special purpose district fields in PLUTO contain a CR% value"
```

Comment:
Reference to row 3: is this correct `variable_type`? It's same as in row 2.

Reply:
fixed